python_training/redis_management.py

```diff
@@ -0,0 +1,36 @@
+import docker
+import redis
+import time
+
+CLIENT = docker.from_env()
+DEFAULT_PORT = 6379
+DEFAULT_CONTAINER_NAME = 'redis'
+IMAGE_NAME = 'redis'
+CONNECTION_ATTEMPTS = 3
+
+
+def run_redis(name=None, port=None):
+    """
+    Runs a redis container and returns a connection to the DB.
+
+    :param: name: The name that would be given to the container.
+    :param: port: The port on localhost that would be bound to the default redis port within the container.
+    :return: A redis connection.
+    """
+    container_name = name if name is not None else DEFAULT_CONTAINER_NAME
```

Comment:
`name or DEFAULT_CONTAINER_NAME`.
But using default values for the function arguments might be easier.